notebooks/neo4j_to_networkx.ipynb

```diff
@@ -0,0 +1,659 @@
+    "]\n",
+    "nx.draw(g6.subgraph([node[0] for node in sabNodes]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "55661449",
+   "metadata": {},
+   "source": [
+    "## Defining subgraph based on page hits from Big Query GA data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "766db8b1",
+   "metadata": {},
+   "source": [
+    "### By page path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "20479711",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>pagePath</th>\n",
+       "      <th>noOfSessions</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>/report-blocked-drain/harlow</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>/government/publications/weekly-statistics-for...</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>/student-finance-calculator/y/[date]/uk-full-t...</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>/government/statistics/suicide-deaths-in-north...</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>/government/publications/psv-speed-limiter-exe...</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                            pagePath noOfSessions\n",
+       "0                       /report-blocked-drain/harlow            5\n",
+       "1  /government/publications/weekly-statistics-for...            5\n",
+       "2  /student-finance-calculator/y/[date]/uk-full-t...            5\n",
+       "3  /government/statistics/suicide-deaths-in-north...            5\n",
+       "4  /government/publications/psv-speed-limiter-exe...            5"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Import the data from google sheets\n",
+    "spreadsheet_key = '1ZIhAuP_PwITRCqrks5zag__JJ6j4gwVmmeQoKdPePI4' \n",
```

Comment:
We discussed that the new plan was to get rid of the code where we import data via the google API, since it's a faff to set up. Instead, we just ask people to download the data folder from our G drive. So then they can write something like pd.read_csv('../data/page_path.csv'). So I'll do that